### ISSUE TITLE
Pronounciation problems with NVDA 2016.2 in French

### ISSUE THREAD

**Issue by DataTriny** [nvaccess/nvda#6038]
Hi, in the latest NVDA 2016.2 version, with French language, the new eSpeak NG fails to pronounce correctly very common things.
Here are two problems I encountered: "votre" and "vôtre" are pronounced exactly the same, "ui" is also not pronounced correctly.
Thanks in advance!

**Comment by michaelDCurran** [comment]
I assume this is not fixed by the latest eSpeakNG master which is now in NVDA Next?

**Comment by michaelDCurran** [comment]
Was this also an issue in 2016.1, or just 2016.2? Has this been reported to the eSpeak-NG French maintainers?

**Comment by DataTriny** [comment]
Problems appeared in NVDA 2016.2 and an issue has been opened on eSpeak-NG repo.

Arnold

**Comment by michaelDCurran** [comment]
Could you please quote the espeak-ng issue number here, so we can track 
it further?


Thanks.

Mick



On 12/07/2016 7:40 PM, Arnold Loubriat wrote:
> Problems appeared in NVDA 2016.2 and an issue has been opened on 
> eSpeak-NG repo.
>
> Arnold
>
> > Le 12 juil. 2016 à 04:06, Michael Curran <notifications@github.com> 
> a écrit :
> >
> > Was this also an issue in 2016.1, or just 2016.2? Has this been 
> reported to the eSpeak-NG French maintainers?
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub, or mute the thread.
> >
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub 
> <https://github.com/nvaccess/nvda/issues/6038#issuecomment-231989706>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ANf5nfEtOJ1M2_YuSeB3Hgw3HLts9maCks5qU2ElgaJpZM4ItzyM>.
>

-- 
Michael Curran
Executive Director, NV Access Limited
Phone: +61 7 3149 3306
Website: http://www.nvaccess.org/
Twitter: @NVAccess
Facebook: http://www.facebook.com/NVAccess

**Comment by bhavyashah** [comment]
Since https://github.com/espeak-ng/espeak-ng/issues/133 has already been closed, can we close here as well? @ehollig

**Comment by ehollig** [comment]
Closing, as it seems https://github.com/espeak-ng/espeak-ng/issues/133 has been resolved.
